bedwars-plugin/src/main/java/com/tomkeuper/bedwars/BedWars.java

```diff
@@ -602,7 +602,7 @@ public void onEnable() {
 
         // Initialize the addons
         Bukkit.getScheduler().runTaskLater(this, () -> {
-            (new AddonManager()).registerAddons();
+            (new AddonManager()).loadAddons();
```

Comment:
I dont really get the "(new AddonManager())" part, cant you just do this.addonManager?

Reply:
True